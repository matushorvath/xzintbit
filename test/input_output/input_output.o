.C
109,27,104,65,203,0,204,0,104,97,104,66,203,0,203,1,204,1,204,0,203,0,204,0,104,98,99,0,0
.R
1
.I
.E
.S
data:27;1
