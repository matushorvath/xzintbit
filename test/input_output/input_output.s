    arb data

    out 'A'
    in  [rb + 0]
    out [rb + 0]
    out 'a'

    out 'B'
    in  [rb + 0]
    in  [rb + 1]
    out [rb + 1]
    out [rb + 0]
    in  [rb + 0]
    out [rb + 0]
    out 'b'

    hlt

data:
    ds 2, 0

.EOF
