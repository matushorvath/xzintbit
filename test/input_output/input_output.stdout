A1aB324b